modules/ROOT/pages/logging-in-mule.adoc

```diff
@@ -0,0 +1,287 @@
+= Configuring Logging
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: mule, studio, logger, logs, log, notifications, errors, debug
+
+Mule automatically logs multiple messages and specific elements in your application flows to help you debug and keep track of events. You can also include the xref:logger-component-reference.adoc[`<logger>` Element] anywhere in a flow and set it up to output any message you want.
+
+You can create a configuration file to define:
+
+ * What kinds of messages to log
+ * How to log the messages (asynchronously or synchronously)
+ * Where to log the messages, such as to the console or disk, or to an endpoint or database.
+
+Mule uses http:ç, which is a logging facade that discovers and uses a logging strategy from the classpath, such as Apache Log4j 2 or the JDK Logger. By default, Mule includes Log4j 2, which is configured with a file called `log4j2.xml`.
+
+The Mule server has a `log4j2.xml` in its `conf` directory, which you can customize when running the server in standalone mode. This configuration file is relevant for data output to the `mule_ee.log` file.
+
+If you remove the `çlog4j2.xml` file from your application's classpath, Mule Runtime applies a default logging configuration to the application and outputs the log to `MULE_HOME/logs/<application-name>.log`.
+
+== Viewing Logs
+To view Anypoint Studio logs:
+
+. Click *Anypoint Studio* > *About Anypoint Studio* > *Installation Details* > *Configuration* tab > *View Error Log*.
++
+image::logging-in-mule-view-error-log.png[View Error Log]
+
+The output from the View Error Log is in the workspace's `.metadata/.log` file, for example:
+----
+/Users/me/AnypointStudio/workspace/.metadata/.log
+----
+
+After you run your Mule application, the log output is in the workspace's `.mule/logs/<project>.log` file.
+For example, for project `abcd`:
+
+----
+/Users/me/AnypointStudio/workspace/.mule/logs/abcd.log
+----
+
+== Synchronous Versus Asynchronous Logging
+
+[NOTE]
+By default, Mule logs messages asynchronously in version 3.6.0 and later.
+
+When logging synchronously, the execution of the thread that is processing your message is interrupted to wait for the log message to be fully handled before it can continue:
+
+image::logger-synch.jpg[logger+synch,align="center"]
```

Comment:
This image is broken.

Reply:
The image looks fine in my build. I will contact you about this.